libs/blocks/global-navigation/global-navigation.js

```diff
@@ -337,7 +334,6 @@ class Gnav {
   };
 
   decorateAppLauncher = () => {
-    // TODO: review App Launcher component
```

Comment:
We actually do need to review this, albeit after MVP. We do have a story for it, so I guess it's ok to remove the comment for now 🤷

Reply:
Correct, this is something I mentioned in the description, we can handle the app launcher as part of the upcoming story (post mvp).